docs/howto/openstack/nova/new-server.md

```diff
@@ -0,0 +1,337 @@
+	| nordostbahnhof |
+	+----------------+
+	```
+		
+	The `ext-net` is an external network (type `openstack network 
+	show ext-net` for more information). You want to connect your server to 
+	an internal network, so set the `NETWORK_NAME` variable accordingly:
+	
+	```bash
+	NETWORK_NAME="nordostbahnhof"
+	```
+		
+	Regarding the security group (`SEC_GROUP_NAME`), unless you 
+	have already created one yourself, you will find only one per region:
+	
+	```bash
+	openstack security group list -c Name -c Description
+	```
+		
+	```plain
+	+---------+------------------------+
+	| Name    | Description            |
+	+---------+------------------------+
+	| default | Default security group |
+	+---------+------------------------+
+	```
+		
+	Go ahead and set `SEC_GROUP_NAME`:
+		
+	```bash
+	SEC_GROUP_NAME="default"
+	```
+		
+	You most likely want a server you can remotely connect to via 
+	SSH without typing a password. Upload one of our public keys to your 
+	{{extra.brand}} Cloud account:
+		
+	```bash
+	openstack keypair create --public-key ~/.ssh/id_ed25519.pub bahnhof
+	```
+		
+	In the example above, we uploaded the public key 
+	`~/.ssh/id_ed25519.pub` to our {{extra.brand}} Cloud account and named 
+	it `bahnhof`. Follow our example and do not forget to set the 
+	`KEY_NAME`:
+		
+	```bash
+	KEY_NAME="bahnhof"
+	```
+		
+	By the way, check all uploaded public keys...
+		
+	```bash
+	openstack keypair list
+	```
+		
+	...and get more information regarding the one you just uploaded:
+		
+	```bash
+	openstack keypair show bahnhof
+	```
+		
+	You are almost ready to create your new server. Decide on a 
+	name...
+		
+	```bash
+	SERVER_NAME="zug" # just an example
+	```
+		
+	...and then go ahead and create it:
+			
+	```bash
+	openstack server create \
+		--flavor 1C-1GB-20GB \
+		--image $IMAGE_NAME \
+		--network nordostbahnhof \
+		--security-group default \
+		--key-name bahnhof \
+		--wait \
+		zug
+	```
+		
+	(For clarity's sake, and with the exception of `IMAGE_NAME`, we 
+	used the actual values and not the variables we so meticulously set.) 
+	By the way, the `--wait` parameter is optional. Whenever you choose to 
+	use it, you get back control of your terminal only after the server is 
+	readily available in {{extra.brand}} Cloud.
+		
+	To see all available servers in the region, type:
+		
+	```bash
+	openstack server list
+	```
+		
+	You can always get specific information on a particular server:
+	
+	```bash
+	openstack server show zug
+	```
+		
+	To connect to your server remotely, you first have to create a 
+	floating IP for the external network in the {{extra.brand}} Cloud, and 
+	then assign this IP to your server. First, create the floating IP: 
+	
+	```bash
+	openstack floating ip create ext-net
+	```
+		
+	See all floating IPs...
+		
+	```bash
+	openstack floating ip list
+	```
+		
+	...and assign the one you just created to your server:
+		
+	```bash
+	openstack server add floating ip zug aaa.bbb.ccc.ddd
```

Comment:
Here, instead of the placeholder `aaa.bbb.ccc.ddd`, you can use a real-looking IPv4 address that is actually one that has been reserved for documentation purposes. See #43.

Reply:
Very nice!